src/Core/MailTemplates/Handlebars/SecretsManagerAccessRequest.html.hbs

```diff
@@ -0,0 +1,27 @@
+{{#>FullHtmlLayout}}
+
+<table width="100%" cellpadding="0" cellspacing="0" style="margin: 0; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
+    <tr style="margin: 0; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
+        <td class="content-block" style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; margin: 0; -webkit-font-smoothing: antialiased; padding: 0; -webkit-text-size-adjust: none;" valign="top">
+            {{UserNameRequestingAccess}} has requested access to secrets manager: <br /><br />
+           <pre style="overflow-x: auto; background-color: #ECECEC; max-width: 700px; overflow-wrap: break-word; border-radius: 10px; padding: 1em; margin-bottom: 2em; ">{{EmailContent}} - {{UserNameRequestingAccess}}</pre>
+        </td>
+        <br/>
+    </tr>
+    <tr style="margin: 0; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
+        <td class="content-block" style="margin-bottom:1em; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; color: #333; line-height: 25px; margin: 0; -webkit-font-smoothing: antialiased; padding: 0; -webkit-text-size-adjust: none;" valign="top" align="left">
+            <a href="https://bitwarden.com/contact-sales/?utm_source=sm_request_access_email&utm_medium=email" clicktracking=off target="_blank" rel="noopener" style="color: #ffffff; text-decoration: none; text-align: center; cursor: pointer; display: inline-block; border-radius: 5px; background-color: #175DDC; border-color: #175DDC; border-style: solid; border-width: 10px 20px; margin: 0; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 16px; line-height: 25px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
```

Comment:
the problem with this link and message (`Contact Bitwarden`) is that it's not really actionable. As an admin of an org receiving that message from a user who might have a valid case for SM, if i decide to explore what is SM or decide to enable it, it would be much better to provide a link to help to get started, instead of contact sales https://bitwarden.com/help/secrets-manager-quick-start/

Reply:
@mzieniukbw the marketing team decides where the button will take them. In my opinion it seems like the marketing team has a good idea to send them to the contact-sales page, as this will allow us to identify that there is a potential lead/revenue source that we can obtain if we help them with their setup.